recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - opt_einsum >=2.3.2
     - pyro-api >=0.1.1
     - python >=3.7
-    - pytorch >=1.11
+    - pytorch >=2.0.1
```

Comment:
do you actually need the 2.0.1 release, or is 2.0.0 enough?

Reply:
I just mirror the requirements from [the Pyro repo](https://github.com/pyro-ppl/pyro/blob/dev/setup.py#L98-L107) and that says 2.0.1 so I say 2.0.1

Reply:
yeah, i saw the PR that did that too:
https://github.com/pyro-ppl/pyro/pull/3192

it feels "safe" to reduce the requirement to 2.0.0 through a patch (so that pip check passes) unless you want to try to wait for 2.0.1 to get compiled.

Reply:
2.0.1 might be a day or two at this point:
https://github.com/conda-forge/pytorch-cpu-feedstock/pull/176#issuecomment-1647061672